service/src/test/java/uk/gov/hmcts/reform/fpl/service/ChildrenServiceTest.java

```diff
@@ -87,96 +85,7 @@ void shouldReturnTrueWhenAllChildrenHaveFinalOrder() {
-                .containsExactly("Yes", "Yes", "Yes");
-
-            assertThat(result).extracting(child -> child.getValue().getFinalOrderIssuedType())
-                .containsOnly("Education supervision order (C37)");
-        }
-
-        @Test
-        void shouldUpdateFinalOrderIssuedWhenAppliesToSelectedChildren() {
-            List<Element<Child>> children = testChildren();
-
-            Selector childSelector = Selector.builder()
-                .count("1")
-                .selected(List.of(1))
-                .build();
-
-            List<Element<Child>> result = service.updateFinalOrderIssued(
-                orderOfType(CARE_ORDER), children, "No", childSelector, null
-            );
-
-            assertThat(result).extracting(element -> element.getValue().getFinalOrderIssued())
-                .containsExactly("No", "Yes", "No");
-
-            assertThat(result).extracting(element -> element.getValue().getFinalOrderIssuedType())
-                .containsExactly(null, "Care order", null);
-        }
-
-        @Test
-        void shouldUpdateFinalOrderIssuedWhenAppliesToSelectedChildrenAndAlreadyIssuedForOtherChildren() {
-            List<Element<Child>> children = List.of(childWithoutFinalOrderIssued(),
-                childWithFinalOrderIssued(),
-                childWithoutFinalOrderIssued(), childWithoutFinalOrderIssued(), childWithoutFinalOrderIssued());
-
-            Selector childSelector = Selector.builder()
-                .count("5")
-                .selected(List.of(0, 2))
-                .build();
-
-            List<Element<Child>> result = service.updateFinalOrderIssued(
-                orderOfType(CARE_ORDER), children, "No", childSelector, null
-            );
-
-            assertThat(result).extracting(element -> element.getValue().getFinalOrderIssued())
-                .containsExactly("Yes", "Yes", "Yes", "No", "No");
-
-            assertThat(result).extracting(element -> element.getValue().getFinalOrderIssuedType())
-                .containsExactly("Care order", "Care order", "Care order", null, null);
-        }
-
-        @Test
-        void shouldUpdateFinalOrderIssuedWhenAppliesToSelectedChildrenAndOneRemainingChild() {
-            List<Element<Child>> children = List.of(childWithFinalOrderIssued(),
-                childWithoutFinalOrderIssued(),
-                childWithFinalOrderIssued());
-
-            List<Element<Child>> result = service.updateFinalOrderIssued(
-                orderOfType(SUPERVISION_ORDER), children, "No", null, "1"
-            );
-
-            assertThat(result).extracting(element -> element.getValue().getFinalOrderIssued())
-                .containsExactly("Yes", "Yes", "Yes");
-
-            assertThat(result).extracting(element -> element.getValue().getFinalOrderIssuedType())
-                .containsExactly("Care order", "Supervision order", "Care order");
-        }
-    }
-
-    @Nested
```

Comment:
Do you no longer need the tests above?

Reply:
I don't they are necessary as I have updated one of the `updateFinalOrderIssued` functions to accept a String so we did not have two very similar functions. The removed tests would not really be testing anything different for the function